.github/workflows/workflow-httpstan.yml

```diff
@@ -0,0 +1,290 @@
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          python -m pip install --upgrade --no-cache-dir pip
+          python -m pip install --no-cache-dir "poetry<2,>=1.0"
+          python -m pip install --no-cache-dir "mypy-protobuf~=1.0"
+      - name: Build libraries
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          make
+      - name: Build wheel
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          python -m poetry run pip install --no-cache-dir Cython
+          python -m poetry build -v
+      - name: Fix wheel
+        env:
+          CC: ${{ matrix.compiler }}
+          PLAT: manylinux2014_x86_64
+        run: |
+          for whl in dist/*-linux*.whl; do
+            auditwheel repair "${whl}" --plat "${PLAT}" -w "./dist";
+            rm ${whl};
+          done
+      - uses: actions/upload-artifact@v2
+        with:
+          name: httpstan_wheels
+          path: dist/*.whl
+      - name: Upload wheel to release
+        uses: svenstaro/upload-release-action@v1-release
+        if: success() && startsWith(github.ref, 'refs/tags/v')
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: dist/*.whl
+          tag: ${{ github.ref }}
+          overwrite: true
+          file_glob: true
+      - name: Upload to pypi
+        if: success() && startsWith(github.ref, 'refs/tags/v')
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: |
+          python -m pip install --no-cache-dir twine
+          python -m twine upload dist/* -u ${TWINE_USERNAME} -p ${TWINE_PASSWORD} --skip-existing
+  publish_macos:
+    name: macos wheel
+    needs: [pytest, tox-flake8, tox-format, tox-mypy, tox-docs]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        python-version: [3.7, 3.8]
+        compiler: [clang]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: check C++ compiler and Python
+        run: |
+          clang --version
+          clang++ --version
+          which clang
+          python --version
+          which python
+      - name: Install dependencies
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          python -m pip install --no-cache-dir --upgrade pip
+          python -m pip install --no-cache-dir "poetry<2,>=1.0"
+          python -m pip install --no-cache-dir "mypy-protobuf~=1.0"
+      - name: Build libraries
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          make
+      - name: Build wheel
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          python -m poetry run pip install --no-cache-dir Cython
+          python -m poetry build -v
```

Comment:
We need to run tests against the wheel before uploading it.

Reply:
Yes, this is important.

I think we can create a new virtual env and install wheel on it.